elyra/pipeline/processor_airflow.py

```diff
@@ -216,6 +244,27 @@ def _cc_pipeline(self, pipeline, pipeline_name):
                                                           operation)
 
             else:
+                # Change value of variables according to their type. String variables must include
+                # quotation marks in order to render properly in the jinja template and dictionary
+                # values must be converted from strings.
+                component = self._component_registry.get_component(operation.classifier)
+                for property in component.properties:
```

Comment:
We should use a different name here other than `property` as it conflicts with the built-in string.